config/genesis/genesis_test.go

```diff
@@ -0,0 +1,28 @@
+package genesis
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestParseGenesisJson(t *testing.T) {
+	file := "../../genesis.json"
```

Comment:
Lets use a temp file

Reply:
changed to a test genesis file

Reply:
just remembered the code you sent... would that be preferred over just reading the file?

Reply:
Yes that would be preferred over just reading the file.

Reply:
ok done